lstchain/calib/camera/time_sampling_correction.py

```diff
@@ -0,0 +1,118 @@
+import h5py
+import numpy as np
+
+
+from ctapipe.core import Component
+from ctapipe.core.traits import Unicode
+
+__all__ = [
+    'TimeSamplingCorrection',
+    ]
+
+
+
+class TimeSamplingCorrection(Component):
+    """
+        The PulseTimeCorrection class to correct time pulse
+        using Fourier series expansion.
+    """
+
+    time_sampling_correction_path = Unicode(
+        '',
+        help='Path to the waveform sampling correction file',
+        allow_none = True,
+    ).tag(config=True)
+
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+
+        self.time_sampling_coefficients = None
+
+        self.load_sampling_coefficient_file()
+
+    def load_sampling_coefficient_file(self):
+        """
+            Function to load sampling coefficient file.
+        """
+
+        try:
+            with h5py.File(self.time_sampling_correction_path, 'r') as hf:
+                self.time_sampling_coefficients = np.array(hf['sampling_interval_coefficient'])
+        except:
+            self.log.error(f"Problem in reading sampling coefficient file {self.time_sampling_correction_path}")
+
+
+    def get_corrections(self, event, telid):
+        """
+        Get the time/charge sampling corrections for one event and one telescope
+
+        Parameters
+        ----------
+        event: general event container
+        telid: id of the telescope
+
+        Return
+        ------
+        sampling_corrections: np.array (n_gains, n_pixels, n_samples) with the correction factors
+        """
+
+        n_gains = 2
```

Comment:
This is the `nth` location where these numbers are hard coded. These should be defined as module constants somewhere in a common place. 

e.g. `calib/camera/constants.py`

Reply:
Yes, indeed. All these constants will be soon in ctapipe_io_lst. I wuold suggest to leave things as they and there and clean everything with the coming lstchain version crappie 10 compatible